### ISSUE TITLE
New class of POB messages for validators

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1147]
It appears that transcribers are daunted by all of the POB warning messages that are created for the benefit of validators.
It has been proposed that POB warning messages be placed in a category of their own.
Likely a size 8 .Would require a new class of messages; hence lots of donkey work where each message has to be looked at and a decision made as to which class to assign it and then a relay out of the pages to accommodate another class.

Geoff gives Priority Low then. I was hoping that it was just removal of some items from the existing report.

Afraid not. The system was built on 3 messages classes and an attempt to divide 1 into 2 after the fact is a kludge which would introduce maintainability issues for the next programmer. Hence one is far better off to say there are now 4 not 3 and a half

**Comment by geoffj-FUG** [comment]
Summary
This is what I see as the situation:
There are three reports allowed.
Info: this is the least useful but there is a lot of information in it that is useful for specific situations such as new transcribers.
Warnings: This is essential for Validators
Errors: This is essential for Transcribers and Proofreaders as we cannot progress a transcription whilst it contains Errors.
In an ideal world we would have a fourth report which would be Essential Warnings. It would contain all the things that need to be looked at for quality purposes. It would not have the POB Warnings but would contain all the others. However this is not an ideal world.
The only solution that I can see is to expand the Errors report to contain these essential Warnings. This would then be similar, but not exactly the same, as a report from CSVCheck.
As Kirk idetifies this affects the download of a vld file and replacement in FC2. The current situation is that a downloaded vld file needs to be re-validated, which is a waste of time and effort as it has already been done once.
Geoff

**Comment by FreecenBren** [comment]
In my opinion I agree with Geoff when he sums it up quite nicely as:



*The only solution that I can see is to expand the Errors report to contain
these essential Warnings. This would then be similar, but not exactly the
same, as a report from CSVCheck.As Kirk idetifies this affects the download
of a vld file and replacement in FC2. The current situation is that a
downloaded vld file needs to be re-validated, which is a waste of time and
effort as it has already been done once.Geoff*
Especially as it took 20 odd years to get it all online. It could  then
take as long again to redo them. Not an option as far as I can see.

Brenda

**Comment by geoffj-FUG** [comment]
Brenda

 

It is not proposed to download every file. There is a need to download a vld file when we receive a query from a researcher and we need to amend the original transcription.

 

Geoff

**Comment by Captainkirkdawson** [comment]
@geoffj-FUG Amount of effort to do as suggested is 90% of the effort required to add the new class

**Comment by Captainkirkdawson** [comment]
@geoffj-FUG 
As I had to address the issue of the . in Rel or Dis fields in the processing of a traditional header; which is now deployed to test3 and permits with or without the . on those fields.
I decided to deal with this issue at the same time.
There are now four processing options
![POB.png](https://images.zenhubusercontent.com/5b68ae684b5806bc2bdefcf0/473ca1a8-6f05-4f49-a50b-b8c0f1a1d835)

I have set the default to No POB Warnings and suggest this should be the standard option used by transcribers. Validators should use All Warnings to see the POB warnings as well as any others.
Running on test3
